easybuild/easyconfigs/r/RNA-SeQC/RNA-SeQC-2.4.2-foss-2021a.eb

```diff
@@ -0,0 +1,68 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'RNA-SeQC'
+version = '2.4.2'
+versionsuffix = ''
```

Comment:
@branfosj not sure why this is here?

Reply:
Removed in d515d5f

The previous version had a versionsuffix and I accidentally left that bit in.

Reply:
Thanks.